src/Umbraco.Web/Models/Mapping/ContentModelMapper.cs

```diff
@@ -142,9 +142,33 @@ private static void AfterMap(IContent content, ContentItemDisplay display, IData
                 TabsAndPropertiesResolver.AddListView(display, "content", dataTypeService);
             }
 
+            var currentDocumentType = contentTypeService.GetContentType(display.ContentTypeAlias);
+            var currentDocumentTypeName = currentDocumentType == null ? String.Empty : currentDocumentType.Name;
+            var docTypeValue = String.Empty;
+
+#if (DEBUG)
+            if (canAccessUser)
+            {
+                var currentUser = userService.GetUserById(UmbracoContext.Current.Security.CurrentUser.Id);
+                if (currentUser.AllowedSections.Any(x => x.Equals("settings")))

```

Comment:
Instead of hard coding the "settings" string, its better to use Umbraco.Core.Constants.Applications.Settings

I am reviewing and modifying this PR so no need to take any actions.